inc/Dependencies/Database/Query.php

```diff
@@ -2046,7 +2046,7 @@ private function validate_item( $item = array() ) {
 
 			// Strip slashes from all strings
 			if ( is_string( $value ) ) {
-				$value = stripslashes( $value );
+				//$value = stripslashes( $value );// We removed this line at PR #3847 to solve if the content has backslash.
 			}
```

Comment:
Should we comment the whole conditional? No need to check for the type if we're not doing anything with it.

Reply:
Done